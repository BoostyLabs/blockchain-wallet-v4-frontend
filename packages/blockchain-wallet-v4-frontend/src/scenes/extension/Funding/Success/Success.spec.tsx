import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { Success } from '.'

describe('Success', (): void => {
  it('renders correctly', () => {
    const component = shallow(<Success />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
