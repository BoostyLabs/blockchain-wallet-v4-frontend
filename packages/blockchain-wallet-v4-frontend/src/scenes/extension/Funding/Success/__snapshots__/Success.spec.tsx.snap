// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success renders correctly 1`] = `
<Fragment>
  <styled.div>
    <Memo()
      color="blue300"
      label="icon Check"
      size="lg"
    >
      <SvgIconCheck />
    </Memo()>
    <Styled(Text)>
      Success! 
      <br />
      Your funds are now available
    </Styled(Text)>
  </styled.div>
  <Styled(Link)
    style={
      Object {
        "width": "unset",
      }
    }
    to="/extension/home"
  >
    Continue
  </Styled(Link)>
</Fragment>
`;
